core/src/main/scala/com/github/andyglow/json/JsonFormatter.scala

```diff
@@ -14,15 +14,18 @@ class JsonFormatter(step: Int = 2) {
   private def _format(value: Value, sb: StringBuilder, indent: Int = 0): Unit = {
     val i0 = " " * indent
     val i1 = " " * (indent + step)
+    val nl = System.lineSeparator()
```

Comment:
can we have it under the companion object umbrella?
like
```
class JsonFormatter(...) {
  import JsonFormatter._
  
  ...
}

object JsonFormatter extends JsonFormatter(...) {
  val nl = System.lineSeparator()
}

Reply:
Done